LICENSE

```diff
@@ -246,6 +252,9 @@ https://github.com/grantr/elasticsearch-srv-discovery
 developed by Grant Rodgers
 License: MIT License, http://www.opensource.org/licenses/mit-license.php
 
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
```

Comment:
@mxm

Reply:
Yes, I can find code of this in our code base.

Reply:
This is now part of Elasticsearch and used as a plugin, so we can remove this.

Reply:
@mxm did you mean you cannot or you can? your two messages seem contradictory? :)

Reply:
okay thanks!